docs/nuget-org/Deprecate-packages.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Deprecating packages on nuget.org
+description: Detailed description on the process of deprecating packages and how the clients shows this information
+author: anangaur
+ms.author: anangaur
+ms.date: 09/23/2019
+ms.topic: conceptual
+ms.reviewer: karann-msft
+---
+
+# Deprecating packages
+
+As a package publisher, if you no longer want to maintain a package or would like to move your package consumers to another package for various reasons, you can now deprecate the package. 
+
+Package deprecation is different than **unlisting** your package as explained below:
+* **Unlisting** a package prevents its discovery as its hidden in search results. Hence it prevents new installs of the package.
```

Comment:
```suggestion
* **Unlisting** a package prevents its discovery because it is hidden in search results. This prevents new installs of the package.
```